app/models/record.rb

```diff
@@ -172,6 +172,9 @@ def get_column(column_name)
     end
   end
 
+  def version
```

Comment:
I added a version property to the record model class to make it easier to pull out the version, the existing implementation in the erb file had a major flaw once I added additional records with feedback=true.